neon-dumper-plugin/src/postgres_client.rs

```diff
@@ -900,49 +932,63 @@ impl PostgresClient for SimplePostgresClient {
 
     fn log_transaction(
         &mut self,
-        transaction_log_info: LogTransactionRequest,
+        transaction_log_info: &LogTransactionRequest,
     ) -> Result<(), GeyserPluginError> {
         self.log_transaction_impl(transaction_log_info)
     }
 
     fn update_block_metadata(
         &mut self,
-        block_info: UpdateBlockMetadataRequest,
+        block_info: &UpdateBlockMetadataRequest,
     ) -> Result<(), GeyserPluginError> {
         self.update_block_metadata_impl(block_info)
     }
 }
 
+#[derive(Clone)]
 struct UpdateAccountRequest {
     account: DbAccountInfo,
     is_startup: bool,
 }
 
+#[derive(Clone)]
 struct UpdateSlotRequest {
     slot: u64,
     parent: Option<u64>,
     slot_status: SlotStatus,
 }
 
+#[derive(Clone)]
 pub struct UpdateBlockMetadataRequest {
     pub block_info: DbBlockInfo,
 }
 
 #[warn(clippy::large_enum_variant)]
+#[derive(Clone)]
 enum DbWorkItem {
     UpdateAccount(Box<UpdateAccountRequest>),
     UpdateSlot(Box<UpdateSlotRequest>),
     LogTransaction(Box<LogTransactionRequest>),
     UpdateBlockMetadata(Box<UpdateBlockMetadataRequest>),
 }
 
+const RECONNECT_RETRIES_DEFAULT: u32 = u32::MAX;
+const RECONNECT_RETRY_INTERVAL_DEFAULT: u16 = 10;
+const WORK_POSTPONE_INTERVAL_SEC_DEFAULT: u64 = 10;
```

Comment:
You can consider default values for arguments that are not in the config file without using the Option.